app/containers/SendTips/saga.js

```diff
@@ -173,6 +177,44 @@ export function* sendTipsWorker({ resetForm, val, questionId, answerId }) {
   }
 }
 
+export function* sendTipsNotificationWorker({
+  data,
+  questionId,
+  answerId,
+  transactionId,
+  block,
+}) {
+  try {
+    let attempts = 1;
+    while (attempts <= 5) {
```

Comment:
Move 5 into a variable to not hardcode it twice

Reply:
fixed